tests/riotboot_hdr/Makefile

```diff
@@ -0,0 +1,8 @@
+include ../Makefile.tests_common
+
+USEMODULE += riotboot_hdr
+USEMODULE += embunit
+
+CFLAGS += -DLOG_LEVEL=LOG_NONE
```

Comment:
This seems useless in a test application where one would expect a good logging level (DEBUG) with everything displayed.

Reply:
Not if the test application also should showcase the size benefits ;-).

Reply:
It's a leftover from the main use case (riotboot).

I'll remove it. Just note that without this, the booloader size would increase dramatically due to the need of a `printf()` function.

Reply:
But maybe make it overridable, so people can modify it when it is required?

Reply:
> Not if the test application also should showcase the size benefits ;-)

I think here anyways the size doesn't matter that much.

>But maybe make it overridable, so people can modify it when it is required?

However I agree with this, I'll make it this way.

Reply:
I don't really see the use of this. Why would anyone ever not have the output in the test?
As @kYc0o noted, the code is partially shared between the bootloader, the RIOT library code and the header generation code (which runs on host).

The bootloader needs to be as minimal as possible and has otherwise *no* output in order to drop any dependency to the (giant) newlib stdio code. That's why it is compiled with LOG_LEVEL=LOG_NONE. This test application doesn't really benefit from it.

Anyhow, I'll open a removal PR once this is in.

Reply:
If you don't want to make it configurable, then remove the log messages completely. Otherwise, they make no sense.

Reply:
The log messages are as configurable as anywhere else, just compile the corresponding module with "-DLOG_LEVEL=loglevel".

The question here is whether the test case needs two extra makefile lines where developers can set the log level, as if that has any extra meaning for this particular test.

The log messages are interesting if the code is used as a RIOT library or for the header tools. It cannot be compiled in when used in the bootloader for size reasons. That's all pretty much irrelevant for the test.